scripts/verify_fideslang_2_data_migration.py

```diff
@@ -299,7 +306,12 @@ def verify_migration(server_url: str, auth_header: Dict[str, str]) -> None:
     server_orphaned_category: fideslang.models.DataCategory = partial_get(
         resource_key="user.behavior.custom", resource_type="data_category"
     )
-    assert server_orphaned_category, server_orphaned_category
+    assert server_orphaned_category
```

Comment:
the `assert server_orphaned_category, server_orphaned_category` originally seemed like a typo, unless this is doing something i don't understand :)

Reply:
decided to revert this since i was unable to verify the verify script locally, and i see more risk in breaking/confusing this script accidentally vs. covering my very small change!